### ISSUE TITLE
Create comment type independent of content types

### ISSUE THREAD

**Issue by mikemccaffrey** [backdrop/backdrop-issues#2049]
Right now comments are fieldable entities, but the bundle for a comment is constrained by the content type they are attached to. Instead, we should be able to specify an arbitrary number of comment types, and attach each type to one or more entity type and bundle. 

Related to #2048.

**Comment by jenlampton** [comment]
I agree, the current architecture of comments (and how they are tied to individual node bundles) seems completely unnecessary.

I'd love to see a single comment type in core (attached to all node types) with perhaps a contrib solution to create any number of additional comment types.
